react/package.json

```diff
@@ -6,6 +6,8 @@
   "dependencies": {
     "@vtex/css-handles": "^1.0.0",
     "classnames": "^2.2.6",
+    "iframe-resizer": "^4.2.10",
+    "msk": "^1.0.3",
```

Comment:
you are not using this library anymore, right? you can remove this from here and the `react/typings/msk.d.ts` file